documentation/en/cli-lotus-miner.md

```diff
@@ -31,15 +33,19 @@ COMMANDS:
      sealing  interact with sealing pipeline
 
 GLOBAL OPTIONS:
-   --actor value, -a value                  specify other actor to query / manipulate
-   --color                                  use color in display output (default: depends on output being a TTY)
-   --miner-repo value, --storagerepo value  Specify miner repo path. flag(storagerepo) and env(LOTUS_STORAGE_PATH) are DEPRECATION, will REMOVE SOON (default: "~/.lotusminer") [$LOTUS_MINER_PATH, $LOTUS_STORAGE_PATH]
-   --vv                                     enables very verbose mode, useful for debugging the CLI (default: false)
-   --help, -h                               show help
-   --version, -v                            print the version
+   --actor value, -a              specify other actor to query / manipulate
+   --color                        use color in display output (default: depends on output being a TTY)
+   --panic-reports value           (default: "~/.lotusminer") [LOTUS_PANIC_REPORT_PATH]
```

Comment:
ah, this one is missing because `Hidden: true` - better build that into your code too, if a flag is `Hidden`, then exclude it from the list